app/invites/client/admin/adminInvites.js

```diff
@@ -0,0 +1,59 @@
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+import moment from 'moment';
+
+import { t, APIClient } from '../../../utils';
+
+import './adminInvites.html';
+
+Template.adminInvites.helpers({
+	invites() {
+		return Template.instance().invites.get();
+	},
+	daysToExpire() {
+		const { expires, days, createdAt } = this;
+
+		if (days > 0) {
+			if (expires < Date.now()) {
+				return t('Expired');
+			}
+
+			return moment(expires).diff(moment(createdAt), 'days');
+		}
+
+		return t('Never');
+	},
+	maxUsesLeft() {
+		const { maxUses, uses } = this;
+
+		if (maxUses > 0) {
+			if (uses >= maxUses) {
+				return t('None');
+			}
+
+			return maxUses - uses;
+		}
+
+		return t('Unlimited');
+	},
+	creationDate() {
+		const { createdAt } = this;
+
+		return createdAt.toLocaleDateString();
```

Comment:
Use our internal methods to format https://github.com/RocketChat/Rocket.Chat/blob/108ad2177a45637bf57010151dafd5f2e069059c/app/lib/client/lib/formatDate.js#L30